src/components/album/AlbumsList.js

```diff
@@ -84,16 +96,15 @@ const AlbumsList = () => {
 
   //call the function to make API request for search input value
   const handleSearch = () => {
-    fetchAlbums(searchType, searchTerm, limit);
+    fetchAlbums(searchType, searchTerm);
   };
```

Comment:
Rather than calling `fetchAlbums` ourselves, we can just update the `currentPage` to `1`. This will trigger the useEffect on line 69, which will call fetchAlbums, with the appropriate searchType and searchTerm values (based on my review comment left on that line).

```suggestion
  const handleSearch = () => {
    if (currentPage === 1) {
      // resetting the current page to 1 won't trigger useEffect so here we have to call the fetch callback manually
      fetchAlbums(searchType, searchTerm);
    } else {
      setCurrentPage(1);
    }
  };
```

Reply:
After applying this fix, if the user is trying to do search from any page (except the first one) - after first click on SEARCH button he is redirected to the first page, and then he need to press the SEARCH button again to get a search result. We don't have a written requirements, so if this is an expected behavior, I can apply it.

Reply:
After adding all the changes, this fix began to work. However, in this case we are not using the Search button at all! Should we keep it?